INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_del/BAS_repair_mutation_for_del_line_131.sol:BitAseanToken:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "/root/osiris/taintFlow.py", line 652, in perform_taint_analysis
    propagate_taint(taint, tainted_stack, tainted_memory, tainted_storage, instruction, current_stack, previous_block, current_block, next_blocks, arithmetic_errors, sha3_list, false_positives, strings)
  File "/root/osiris/taintFlow.py", line 431, in propagate_taint
    matches = re.compile('Extract\(255, 5, 31 \+ (.+?)\)').findall(remove_line_break_space(arithmetic_error["instruction"].data_in[0]))
  File "/usr/lib/python2.7/re.py", line 194, in compile
    return _compile(pattern, flags)
  File "/usr/lib/python2.7/re.py", line 235, in _compile
    cachekey = (type(key[0]),) + key
  File "/root/osiris/symExec.py", line 2868, in handler
    raise Exception("timeout")
Exception: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 89.5%
INFO:symExec:	  Arithmetic bugs: 	 True
INFO:symExec:	  └> Overflow bugs: 	 True
//root/smartbugs/dataset/int_bugs_9/0604/repair_del/BAS_repair_mutation_for_del_line_131.sol:BitAseanToken:134:9
balanceOf[_to] += _value
^
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.0504841805 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_del/BAS_repair_mutation_for_del_line_131.sol:owned:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 98.6%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 0.137593984604 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_del/BAS_repair_mutation_for_del_line_131.sol:token:
INFO:symExec:Running, please wait...
Exception ctypes.ArgumentError: "argument 2: <type 'exceptions.Exception'>: timeout" in <bound method FuncDeclRef.__del__ of <> ignored
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 94.7%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.2094359398 seconds ---
INFO:symExec:	====== Analysis Completed ======

  .oooooo.             o8o            o8o          
 d8P'  `Y8b            `"'            `"'          
888      888  .oooo.o oooo  oooo d8b oooo   .oooo.o
888      888 d88(  "8 `888  `888""8P `888  d88(  "8
888      888 `"Y88b.   888   888      888  `"Y88b. 
`88b    d88' o.  )88b  888   888      888  o.  )88b
 `Y8bood8P'  8""888P' o888o d888b    o888o 8""888P'


!!! SYMBOLIC EXECUTION TIMEOUT !!!
Unexpected error: <type 'exceptions.Exception'>


!!! SYMBOLIC EXECUTION TIMEOUT !!!